ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/table-export/presentation.tsx

```diff
@@ -48,6 +51,7 @@ export default hot(module)((props: Props) => {
     warning,
     customExportCount,
   } = props
+  let loading = false
   return (
     <div className="w-full h-full overflow-auto p-2">
```

Comment:
Could we try having some spacing around the content in the modal. Is it possible to make the modal medium rather than small?